### PR TITLE
Needs to be changed some code

### PR DESCRIPTION
**Pull request by SHUB9914**
The existing code of the example is not working, I have changed something so please review and confirm it.
First thing Here is creating main by extending App within another object so when we run it, it will through an error which is mentioned below  - 
Error: Could not find or load main class docs.http.scaladsl.server.MyExplicitExceptionHandler.MyApp

and the second thing is no need to define the object within object if we do so then application terminates the execution

### CODE REVIEW

**Comment by jrudolph** [comment]
OK TO TEST

File: docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala [hunk 1]
```diff
@@ -25,7 +25,6 @@ object MyExplicitExceptionHandler {
       }
   }
 
-  object MyApp extends App {
     implicit val system = ActorSystem()
```
Comment (jrudolph) [inline docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala]: Can you also fix the indentation?

**Comment by jrudolph** [comment]
@SHUB9914 this code was not supposed to be run verbatim but only existed there that the basic usage is compiled before being shown in the documentation. But if it is as easy to be completely copy-paste-able, then why not :)

Could you fix the indentation where you removed the extra wrapping object?

**Comment by SHUB9914** [comment]
hi @jrudolph 
I have fixed the indentation

File: docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala (hunk 1 above)
Comment (jrudolph) [reply docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala]: Ah I meant removing that extra space characters in this block (and another one below) that formerly belonged to `MyApp`.

**Comment by SHUB9914** [comment]
removed extra spaces

**Comment by jrudolph** [comment]
If you look at the resulting source code, lines are still a bit misindented because you removed one layer of wrapping:

https://github.com/SHUB9914/akka-http/blob/ac965732e558a670ddfa0b3af30c5c87177c0c85/docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala#L9-L36

**Comment by raboof** [comment]
There's also a merge conflict now.. #1465 seems to have solved the same problem in a slightly different way, I'll merge that one in favor of this one. Nonetheless thanks for looking into this, much appreciated!

**Pull request closed without merging**